picky-asn1-x509/src/name.rs

```diff
@@ -19,8 +19,9 @@ pub enum NameAttr {
     LocalityName,
     StateOrProvinceName,
     StreetName,
-    OrganisationName,
-    OrganisationalUnitName,
+    OrganizationName,
+    OrganizationalUnitName,
+    EmailAddress,
```

Comment:
```suggestion
```
To prevent accidental usage, I would rather not add this variant.
This enum is mostly used for the `Name::add_attr` method and I prefer when invalid values can't be provided.
Since you added `Name::add_email` this should be enough.
What do you think?